### ISSUE TITLE
A series should have its own folder inside the book storage folder.

### ISSUE THREAD

**Issue by Mbucari** [rmcrackan/Libation#124]
The books folder gets too cluttered with podcasts. For episode types, can you make a Series folder in the book storage location and place all episodes inside it?  

EDIT: Actually, this would be nice for all books that belong to a series.

**Comment by rmcrackan** [comment]
I could add yet another bool to the settings to allow this, but I'd rather address it under #50 . I'm currently looking through the roadblocks I've found.

A lot of architecture decisions were made very early on about finding live files and having the id be in the filename. This is back when it was just for me. I wanted to be able to move files and have Libation find them quickly. The first step in changing this paradigm was to store liberated state in db. That's done. The next is to de-claw everywhere that expects to find files based on the id being in the path. This is proving to be messy but I'm sure it's surmountable. I've located about 15 important places in code where this needs to be addressed. Once this is done, the final and easiest part will be implementing the feature to allow users to define their own paths.
